garrysmod/gamemodes/terrortown/entities/weapons/weapon_ttt_wtester.lua

```diff
@@ -107,10 +110,10 @@ function SWEP:PrimaryAttack()
 
    local tr = util.TraceLine({start=spos, endpos=sdest, filter=self.Owner, mask=MASK_SHOT})
    local ent = tr.Entity
-   if IsValid(ent) and (not ent:IsPlayer()) then
+   if IsValid(ent) then
       if SERVER then
-         if ent:IsPlayer() then
-            --self:GatherPlayerSample(ent)
+         if ent:IsPlayer() and scanplayer then

```

Comment:
should be scanplayer:GetBool(), did you test this before creating a pull request?